lib/scholar/covariance.ex

```diff
@@ -29,39 +29,61 @@ defmodule Scholar.Covariance do
   @doc """
   Computes covariance matrix for sample inputs `x`.
 
+  #{~S'''
+  The value on the position $(i, j)$ in matrix is caluculated using formula:
+  $$ Cov(X\_{i}, X\_{j}) = \frac{\sum\_{k}\left(x\_{k} - \bar{x}\right)\left(y\_{k} - \bar{y}\right)}{N - 1} $$
+  Where:
+
+    * $X\_{i}$ is a $i$th row of input
+
+    * $x\_{k}$ is a $k$th value of $X_{i}$
+
+    * $y\_{k}$ is a $k$th value of $X_{j}$
+
+    * $\bar{x}$ is the mean of $X\_{i}$
+
+    * $\bar{y}$ is the mean of $X\_{j}$
+
+    * $N$ is the number of samples
```

Comment:
```suggestion
    * $X_i$ is a $i$th row of input

    * $x_k$ is a $k$th value of $X_i$

    * $y_k$ is a $k$th value of $X_j$

    * $\bar{x}$ is the mean of $X_i$

    * $\bar{y}$ is the mean of $X_j$

    * $N$ is the number of samples
```

Reply:
I've discovered right now that for a single index, the curly brackets are redundant and syntax x_i is fine. 😅